### ISSUE TITLE
Bug Report - AltGr key do not work

### ISSUE THREAD

**Issue by f-quintero** [microsoft/terminal#1798]
# Environment

```none
Windows build number: [run "ver" at a command prompt]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

# Expected behavior

# Actual behavior

**Comment by f-quintero** [comment]
Windows 10, Latinamerican keyboard

**Comment by DHowett-MSFT** [comment]
We recently fixed this with #521 and since you didn’t fill out the template *we don’t know what version you are using*.

**Comment by rik2887** [comment]
Same problem, can't type "@" character in any terminal (PowerShell, cmd or Linux). This is my version:

Microsoft Windows [Version 10.0.18362.207]
